node_modules/gh-calendar/lib/util.js

```diff
@@ -37,13 +37,51 @@ var eventsToICal = module.exports.eventsToICal = function(calendarInfo, events)
 
     // Convert each Grasshopper event to an ical VEvent and add it to the calendar
     _.each(events, function(event) {
+        // Get the organisers as simple strings. Ensure they are sorted so the iCal
+        // feed doesn't change between fetches
```

Comment:
Should we add that the organisers will be added to the description of the event?